### PR TITLE
Implement @RequestParam-based REST Controller

### PR DESCRIPTION
**Pull request by alszla**
Initial implementation

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
 plugins {
     id 'java'
 }
 
+apply plugin: 'org.springframework.boot'
+apply plugin: 'io.spring.dependency-management'
+
+bootJar {
+    baseName = 'webservice'
+    version =  '0.1.0'
```
Comment (JosueRodriguezGarcia) [inline build.gradle]: change the version to 1.0

File: src/main/java/com/fundation/webservice/controller/Greeting.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+ package com.fundation.webservice.controller;
+
+/**
+ * Implements the class to be created from a JSON data received by the webservice
+ *
+ * @author Alejandro Sánchez Luizaga
+ * @version 0.1.0
```
Comment (JosueRodriguezGarcia) [inline src/main/java/com/fundation/webservice/controller/Greeting.java]: Change the version to 1.0

File: src/main/java/com/fundation/webservice/main.java [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
 package com.fundation.webservice;
 
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * Implements a dummy client for testing the REST controller
+ *
+ * @author Alejandro Sánchez Luizaga
+ * @version 0.1.0
```
Comment (JosueRodriguezGarcia) [inline src/main/java/com/fundation/webservice/main.java]: Change the version to 1.0

**Comment by LimbertVargas** [comment]
ok, It's all great Alejandro, good job

**Pull request merged**